CONTRIBUTING.rst

```diff
@@ -0,0 +1,244 @@
+- A developer moves the issue to **In Progress** when work starts on it.
+- As work on an issue nears completion, the developer opens a pull request for the changes and moves the issue to **Review**.
+- Developers and stakeholders review the code, tests, and a demonstration of the changed behavior. Only after everyone is satisfied do we accept the pull request and move the issue to **Closed**.
+- Celebrate as the changes go live! 🎉
+
+
+Coding style
+============
+
+**cloudigrade** is written mostly in Python with support targeting the latest release versions of Python 3.6 and pypy3.
+
+We enforce consistency in the code using `Flake8 <https://pypi.python.org/pypi/flake8>`_ with the following additional plugins enabled:
+
+- `flake8-docstrings <https://pypi.python.org/pypi/flake8-docstrings>`_
+- `flake8-quotes <https://pypi.python.org/pypi/flake8-quotes>`_
+- `flake8-import-order <https://pypi.python.org/pypi/flake8-import-order>`_
+
+This means that all our submitted code conforms to `PEP 8 <https://www.python.org/dev/peps/pep-0008/>`_ standards.
+
+All code **must** cleanly pass Flake8 checks before it can be accepted. Occasional exceptions may be made to skip checks with ``# noqa`` comments, but these are strongly discouraged and must be reasonably justified.
+
+Imports must follow the ``pycharm`` style. If you are using PyCharm as your IDE, you can coerce it to use a compliant behavior by configuring your settings as follows:
+
+.. image:: docs/pycharm-settings-imports.png
+   :alt: editor: code style: imports: optimize imports
+
+Alternatively, you may use the command-line tool `isort <https://pypi.python.org/pypi/isort>`_ which has default settings that match closely enough for **cloudigrade**. `isort` can be used to automatically clean up imports across many files, but please manually review its changes before committing to ensure that there are no unintended side-effects. Example usage:
+
+.. code-block:: bash
+
+    # view diff of suggested changes
+    isort -df -rc ./cloudigrade/
+
+    # apply all changes to files
+    isort -rc ./cloudigrade/
+
+
+Contributing code
+=================
+
+**cloudigrade** code lives on `GitHib <https://github.com/cloudigrade/>`_, and all contributions should be submitted there via pull requests.
+
+
+Branching strategy
+------------------
+
+**cloudigrade** follows a simplified `git flow <http://nvie.com/posts/a-successful-git-branching-model/>`_. The ``master`` branch is production-like and reflects the state of the released/live running service at any time (thanks to continuous deployment). All in-development work lives in other branches. We do *not* have perpetual ``develop`` or ``release`` branches. Changes are introduced to master through pull requests directly from short-lived feature branches.
```

Comment:
It's almost github flow (https://guides.github.com/introduction/flow/) except that we don't currently "deploy" our branches.